packages/dialtone-vue3/components/hovercard/timer.js

```diff
@@ -0,0 +1,61 @@
+import { ref } from 'vue';
+import { TOOLTIP_DELAY_MS } from '@/components/tooltip/index.js';
+
+export default (function () {
+  const current = ref(null);
+  let timerLeave, timerEnter;
+  let start = null;
+  let prevCard = null;
+
+  function enter (id) {
```

Comment:
Maybe some short comment description, it's a bit hard to tell what is going on here without going through it line by line.

Reply:
Fixed in latest changes, let me know if it's clear enough